sources/packages/backend/workflow/test/2025-2026/fulltime-assessment/costs/fulltime-assessment-costs-scholarship-bursary.e2e-spec.ts

```diff
@@ -0,0 +1,197 @@
+import {
+  ZeebeMockedClient,
+  createFakeConsolidatedFulltimeData,
+  executeFullTimeAssessmentForProgramYear,
+} from "../../../test-utils";
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
+
+describe(`E2E Test Workflow part-time-assessment-${PROGRAM_YEAR}-costs-scholarship-bursary.`, () => {
+  it(
+    "Should calculate $0 for total scholarship and bursaries when " +
+      "student declares that they did not receive any scholarship and bursaries.",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedFulltimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.studentDataScholarshipAmount = null;
+      assessmentConsolidatedData.offeringWeeks = 18;
+      assessmentConsolidatedData.studentDataRelationshipStatus = "single";
+      // Act
+      const calculatedAssessment =
+        await executeFullTimeAssessmentForProgramYear(
+          PROGRAM_YEAR,
+          assessmentConsolidatedData,
+        );
+      // Assert
+      // $0 or null when student did not receive any scholarship or bursaries.
+      expect(
+        calculatedAssessment.variables.calculatedDataTotalScholarshipsBursaries,
+      ).toBe(0);
+    },
+  );
+
+  it(
+    "Should calculate $0 for total scholarship and bursaries when " +
+      "student declares that they received scholarship and bursaries less than the program year limit " +
+      "and student has not declared any scholarship or bursaries in the current program year.",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedFulltimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.offeringWeeks = 18;
```

Comment:
Please remove the explicit value set for the `offeringWeeks` unless it is pertinent to the E2E scenario.